src/Main.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+class Main {
+    static int arraySize;
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Podaj rozmiar tablicy liczb całkowitych");
+        arraySize = scanner.nextInt();
+
+        int[] array = new int[arraySize];
+
+        for (int i = 0; i < array.length; i++) {
+            System.out.println("Podaj liczbę: ");
+            int number = scanner.nextInt();
+            array[i] = number;
+        }
+        System.out.println("Dla liczb: ");
+        System.out.println(Arrays.toString(array));
+
+        System.out.println("Liczby uporządkowane to: ");
+        int[] bubbleSort = bubbleSort(array);
+
+        System.out.println(Arrays.toString(bubbleSort));
+
+        ComparatorAndCreate comparator = new ComparatorAndCreate();
+
+        System.out.println("Liczba porównań dla tablicy: " + Arrays.toString(comparator.createTableOneToFive()) +
+                " to:" + comparator.comparisons(comparator.createTableOneToFive()));
+        System.out.println("Liczba porównań dla tablicy: " + Arrays.toString(comparator.createTableFiveToOne()) +
+                " to:" + comparator.comparisons(comparator.createTableFiveToOne()));
+        System.out.println("Liczba porównań dla tablicy: " + printLongTable(comparator.createTableOneHundredToOne()) +
+                " to:" + comparator.comparisons(comparator.createTableOneHundredToOne()));
+        System.out.println("Liczba porównań dla tablicy: " + printLongTable(comparator.createTableOneToNinetyNine()) +
+                " to:" + comparator.comparisons(comparator.createTableOneToNinetyNine()));
+    }
+
+    static String printLongTable(int[] array) {
+        return "[" + array[0] + ", " + array[1] + ", ..., " + array[array.length - 2] + ", "
+                + array[array.length - 1] + "]";
+    }
+
+    static int[] bubbleSort(int[] array) {
```

Comment:
Do tego algorytmu można było jeszcze dodać sprawdzenie czy przy przy iteracji nie było żadnych zmian. Jeśli nie - to znaczy, że tablica została już całkowicie posortowana. Sprawdza się to przy częściowo posortowanych tablicach